### DISCUSSION TITLE
Un account genitore con la possibilità di associare più alunni e del planning stile Argo

### DISCUSSION THREAD

**Discussion by fxlduto** [iisgiua/giuaschool#782] [Idee e proposte]
Chiedo allo staff di implementare la funzione di associare uno o più alunni a più di un genitore e il planning come quello di argo! Grazie.

**Comment by trinko** [comment]
E' già previsto associare più alunni allo stesso genitore. Anche se vengono creati utenti distinti per ogni alunno, basta che risulti lo stesso NOME-COGNOME-CODICEFISCALE per identificare lo stesso genitore.
Al genitore viene fornito un solo username (anche se ne esistono tanti quanti sono i figli): subito dopo l'accesso viene mostrato al genitore una pagina di scelta del profilo del figlio.

Per quanto riguarda l'agenda dovresti chiarire cosa vorresti aggiungere.
